q2_feature_classifier/_cutter.py

```diff
@@ -175,11 +181,14 @@ def extract_reads(sequences: DNASequencesDirectoryFormat, f_primer: str,
     q2_types.DNAFASTAFormat
         containing the reads
     """
-    if min_length > trunc_len - trim_left and trunc_len > 0:
+    if min_length > trunc_len - (trim_left + trim_right) and trunc_len > 0:
         raise ValueError('The minimum length setting is greater than the '
                          'length of the truncated sequences. This will cause '
                          'all sequences to be removed from the dataset. To '
-                         'proceed, set a min_length ≤ trunc_len - trim_left.')
+                         'proceed, set '
+                         '"min_length ≤ trunc_len - (trim_left  + '
```

Comment:
looks like there is an extra " character on this line, probably residue from earlier changes... could you please remove?